src/components/application_manager/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -344,12 +344,12 @@ void SetGlobalPropertiesRequest::on_event(const event_engine::Event& event) {
 
     mobile_apis::Result::eType result_code;
     const char* return_info = NULL;
+    const std::string return_string = "Unsupported phoneme type sent in a prompt";
```

Comment:
@justinjdickow  why you declared ```const std::string return_string``` in this  scope?
You use this string inside if scope.